ci-operator/step-registry/telco5g/ptp/tests/telco5g-ptp-tests-commands.sh

```
@@ -170,7 +170,19 @@ spec:
+    dockercgf=$(oc -n openshift-ptp get secret -o jsonpath='{range .items[*]}{.metadata.name}{"\n"}{end}' | grep '^builder-dockercfg-' || true)
+    if [[ -n "${dockercgf}" ]]; then
+      break
+    fi
+    echo "Waiting for builder-dockercfg secret (attempt ${i}/80)..."
+    sleep 5
+  done
+  if [[ -z "${dockercgf}" ]]; then
+    echo "[ERROR] builder-dockercfg secret not found after 400s"
+    exit 1
+  fi
   jobdefinition=$(sed "s#BUILDER_DOCKERCFG#${dockercgf}#" <<<"$jobdefinition")
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)